CenterForOpenScience/osf.io
# [ENG-1633]change sloan fields from bools to choice fields

## Purpose

There were some fields for sloan that were bool fields that should have been choice fields, this makes them choice fields.

## Changes

- changes Bool fields to textfields
- changes tests
- exposes `in_sloan_study` to the front-end and tests
## QA Notes

1. There's no data migration, so existing preprints that QA has tested already with True or False set on these fields will likely be broken. This should only happen on Staging1, and only with the preprints that QA has already touched, so we probably won't bother with the data migration.
2. Light testing only needed. Just make sure patching these fields sets the proper information, much like with the last round, but with the new strings instead of the booleans.

## Documentation

Not user facing, no docs.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/ENG-1633
